qownnotes/snapcraft.yaml

```diff
@@ -10,7 +10,7 @@ description: |
 
 apps:
   qownnotes:
-    command: desktop-launch usr/bin/QOwnNotes --snap
+    command: desktop-launch ${SNAP}/usr/bin/QOwnNotes --snap
```

Comment:
Is this needed? I think it's not, correct? Actually, `${SNAP}/usr/bin/` should be in $PATH for the snap, so:
`desktop-launch QOwnNotes --snap` should be enough. Mind trying?